Dockerfile

```diff
@@ -1,11 +1,16 @@
 ARG BASE_IMAGE
 FROM $BASE_IMAGE AS compile
 
-# Install ZIP and cURL
-RUN apt-get update && \
-    apt-get install -y zip unzip curl git && \
-    rm -rf /var/lib/apt/lists/* && \
-    rm -rf /tmp/*
+# Install commonly used packages
+RUN apt-get update
+RUN apt-get install -y zip
+RUN apt-get install -y unzip
+RUN apt-get install -y curl
+RUN apt-get install -y git
+
+# Cleanup temp files
+RUN rm -rf /var/lib/apt/lists/*
+RUN rm -rf /tmp/*
```

Comment:
Just a note that this change produces intermediate layers that will be pushed to the repository (increasing the storage size), instead of a single layer that already has all the temp files removed and it's lighter

At least for the cleanup steps, I think they can be kept as a single command, since Lacework won't hint additional vulnerabilities here

Reply:
That's fair. Let me revert that commit.